CHANGES

```diff
@@ -41,6 +41,7 @@ Bugs fixed
 * #9649: HTML search: when objects have the same name but in different domains,
   return all of them as result instead of just one.
 * #9678: linkcheck: file extension was shown twice in warnings
+* Fix bug ``.. code::`` not recognising ``:classes:`` option instead of ``:class:`` option.  The new behaviour is only to accept ``:class:``.
```

Comment:
This is not correct. The `code` directive does not recognize the `:classes:` option. Simply, it ignores given `:class:` option.